python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -367,8 +415,18 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
         # the PSF kernel and checking for an InvalidParameterError.
         try:
             _ = psf.getKernel(psf.getAveragePosition())
-        except pexExcept.InvalidParameterError:
-            raise RuntimeError("Failed to determine psfex psf: too few good stars.")
+        except pexExcept.InvalidParameterError as e:
+            ndim = int(str(e).split('saw ')[1].split()[0])
+            # Algorithmically impossible to determine PSF; raise an error
+            # indicating a generic problem with the entire algorithm that
+            # caused the whole task to fail. Include some metadata for QA.
```

Comment:
This comment doesn't really add anything: the act of raising says all that already.

Reply:
Removed!